.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+<!-- Thanks for creating this bug report! Please try to provide as much information as possible so we can recreate the bug. Also, try searching GitHub issues to see if someone has previously posted a similar bug. -->
+
+### Bug Description
+A clear and concise description of what the bug is.
+
+### How To Reproduce
+Steps to reproduce the behavior.
+
+### Expected Behavior
+A clear and concise description of what you expected to happen.
+
+### Installation Information
+Describe your installation method and system information.
+ - Installation method:
+   - [ ] Binary install using anaconda
+   - [ ] Installation from source using anaconda
+   - [ ] Manual installation without anaconda
+   - [ ] Other: please describe
+ - OS:
+   - [ ] Windows: version?
+   - [ ] Ubuntu: version?
+   - [ ] macOS: version?
+   - [ ] Other: please describe
```

Comment:
Maybe we could replace 'Ubuntu' with 'Linux'? I am also not sure how the interface for the buttons works, if there is a box to easily place the version number after the checkbox. If not, the OS question might not be that useful since the same information will also be contained somewhere else in the issue post (when they describe their version information, it should give away their OS type).

Reply:
There is no interface. This text is simply pre-filled into the text box for issue submission. The user can edit it however they like. I originally just had the question about OS, but thought it would be clearer with options.